_episodes/format.md

```diff
@@ -57,190 +60,215 @@ Tabular data have specific properties that are better supported by the spreadshe
 >
 {: .discussion}
 
+> ## FAIR principles
+>
+> Let's have a look at [FAIR
+> principles](https://www.go-fair.org/fair-principles/). According to the
+> principles, how does a file format improve FAIRness of research data?
+>
+>> ## Solution
+>>
+>> - I1:(meta) data use a formal, accessible, shared, and broadly applicable
+>>   language for knowledge representation. For example, if data is provided in
+>>   commonly understood and preferably open formats.
+>>
+>> - R1.3: (meta)data meet domain-relevant community standards e.g. those for
+>   data formats.
+> {: .solution}
+{: .challenge}
+
 ## Recommended formats for different types
 
-There are several data types like geospatial, tabular, storyline, documentation and paper,
-textual, video/audio, and image/figure.
-Some of them are more in line with the FAIR principles than some others.
-Here are some recommended formats for different types:
-
-- Geospatial data: [NetCDF](https://www.unidata.ucar.edu/software/netcdf/docs/netcdf_data_model.html)
-- Tabular data: [SQLite](https://www.sqlite.org/fileformat.html)
-- Textual data: [Markdown](https://daringfireball.net/projects/markdown/)
-- Image data: [TIFF 6.0 uncompressed](https://www.adobe.io/content/dam/udp/en/open/standards/tiff/TIFF6.pdf)
-- Audio data: [Lossless Audio Codec (FLAC)](https://xiph.org/flac/)
-- Video data: [MPEG-4](https://en.wikipedia.org/wiki/MPEG-4)
-- Documentation: [Microsoft Word](https://en.wikipedia.org/wiki/Microsoft_Word) or [PDF](https://www.iso.org/standard/63534.html)
-- Papers & Articles: [LaTeX](https://www.latex-project.org/)
-- Storyline Definitions: [JSON](https://www.json.org/json-en.html)
+There are several data types like geospatial, tabular, storyline, documentation
+and paper, textual, video/audio, and image/figure. Some of them are more in line
+with the FAIR principles than some others. Here are some recommended formats for
+different types:
+
+<table style="width:40%">
+  <thead>
+  <tr>
+      <th>Data type</th>
+      <th>Data format</th>
+    </tr>
```

Comment:
Wouldn't markdown syntax be simpler for this table? https://idratherbewriting.com/documentation-theme-jekyll/mydoc_tables.html#multimarkdown-tables

Or does that not work?

Reply:
I couldn't change the width of the table using markdown syntax.